modules/service-fips.sh

```diff
@@ -88,9 +91,8 @@ _fips_enabled_check() {
     echo 0
 }
 
-_fips_prep_check() {
-    local fips_hmacs pkg power_cpu_ver
-
+_fips_check_arch() {
+    local power_cpu_ver
```

Comment:
I'm thinking _fips_check_arch() should return 7 in the case of failure, or its call in fips_check_support() should || return 7

Reply:
good catch, done